### PR TITLE
adding boostrap feature

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

**Comment by ykhedher** [comment]
@ivarconr  Shall I keep the code style or restore as as it was before ?

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.5",
-    "tiny-emitter": "^2.1.0"
+    "tiny-emitter": "^2.1.0",
+    "unleash-proxy-client": "^1.5.0"
```
Comment (ivarconr) [inline package.json:47]: The proxy client should not depend on itself.

File: src/index.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-import Metrics from './metrics';
-import type IStorageProvider from './storage-provider';
-import LocalStorageProvider from './storage-provider-local';
-import InMemoryStorageProvider from './storage-provider-inmemory';
+import { TinyEmitter } from "tiny-emitter";
+import Metrics from "./metrics";
+import { boostrapData } from "./boostrapData";
+import type IStorageProvider from "./storage-provider";
+import LocalStorageProvider from "./storage-provider-local";
+import InMemoryStorageProvider from "./storage-provider-inmemory";
```
Comment (ivarconr) [inline src/index.ts:6]: Please use single quote

File: src/index.ts [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 
-const DEFINED_FIELDS = ['userId', 'sessionId', 'remoteAddress'];
+const DEFINED_FIELDS = ["userId", "sessionId", "remoteAddress"];
```
Comment (ivarconr) [inline src/index.ts:8]: Please use single quote

File: src/index.ts [hunk 4]
```diff
@@ -16,258 +17,285 @@ export interface IMutableContext {
     sessionId?: string;
     remoteAddress?: string;
     properties?: {
-        [key:string]: string,
+        [key: string]: string;
```
Comment (ivarconr) [inline src/index.ts:20]: This formatting is breaking or general rules.

**Comment by ivarconr** [comment]
Its a bit hard to review when the formatting is this off. I think we need to fix that first!

**Comment by ivarconr** [comment]
> @ivarconr Shall I keep the code style or restore as as it was before ?

yes please!

File: package.json (hunk 1 above)
Comment (ykhedher) [reply package.json:47]: I think it was added when i run `npm install`

File: src/index.ts (hunk 2 above)
Comment (ykhedher) [reply src/index.ts:6]: That's the prettier effect. I'll change it.

**Comment by ykhedher** [comment]
Check [NEW PR (new branch)](https://github.com/Unleash/unleash-proxy-client-js/pulls)

**Pull request closed without merging**